unattended_installer/install_functions/installCommon.sh

```diff
@@ -270,10 +270,63 @@ function installCommon_getPass() {
     done
 }
 
+function installCommon_installCheckDependencies() {
+
+    if [ "${sys_type}" == "yum" ]; then
+        dependencies=( systemd grep tar coreutils sed procps gawk lsof curl openssl )
```

Comment:
This loop is repeated in multiple places in the script maybe we could create a function to avoid repeating code.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2089/commits/f60dfa04387b4fd109eed128b1ee3019d07659a8.

Added functions `installCommon_aptInstallList` and `installCommon_yumInstallList` which receive a list of dependencies. This functions are used in the installation of the checks and the prerequisites.